src/styles/components/search-input.scss

```diff
@@ -0,0 +1,12 @@
+.search-input {
+  position: relative;
+  width: 100%;
+  display: inline-block;
+
+  &__icon {
```

Comment:
It might be more flexible to define this as a class like `right-content` and it's better to use flexbox than absolute positioning for this.